db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/StatementSchemaPermission.java

```diff
@@ -337,4 +360,35 @@ public String getSchemaName() {
 		return schemaName;
 	}
 
+	/**
+	 * Routine to check if another instance of access descriptor matches this.
```

Comment:
nit: s/Routing to check/Checks

Reply:
Removed as suggested in the previous comment.